test/e2e_node/e2e_node_suite_test.go

```diff
@@ -124,45 +120,47 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	// We should mask locksmithd when provisioning the machine.
 	maskLocksmithdOnCoreos()
 
-	shared := &SharedContext{}
+	var data []byte
 	if *startServices {
-		e2es = newE2eService(framework.TestContext.NodeName, framework.TestContext.CgroupsPerQOS, framework.TestContext.EvictionHard, shared)
-		if err := e2es.start(); err != nil {
+		e2es = NewE2eServices()
```

Comment:
I didn't make myself clear in the previous comment. Since NewE2eServices only returns an empty object, maybe StartE2EServices() can return new the object directly? 

If the new-then-start pattern is preferred, I think the naming should be:
func NewE2eServices() *E2EServices
func (e *E2EServices) Start() (string, error)